common/persistence/dataInterfaces.go

```diff
@@ -1436,6 +1460,7 @@ type (
 		CreateShard(request *CreateShardRequest) error
 		GetShard(request *GetShardRequest) (*GetShardResponse, error)
 		UpdateShard(request *UpdateShardRequest) error
+		ScanWorkflows(request *ScanWorkflowsRequest) (*ScanWorkflowsResponse, error)
```

Comment:
How about calling this ListXXX() to stick with our naming convention of create/read-get/update/delete/list ?